span-normalizer/build.gradle.kts

```diff
@@ -39,17 +39,22 @@ dependencies {
   implementation(project(":span-normalizer-api"))
   implementation(project(":span-normalizer-constants"))
 
-  implementation("org.hypertrace.core.datamodel:data-model:0.1.9")
+  implementation("org.hypertrace.core.datamodel:data-model:0.1.10")
   implementation("org.hypertrace.core.serviceframework:platform-service-framework:0.1.9")
   implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.8")
   implementation("org.hypertrace.core.kafkastreams.framework:kafka-streams-framework:0.1.7")
 
 
   // Required for the GRPC clients.
-  runtimeOnly("io.grpc:grpc-netty:1.31.1")
+  runtimeOnly("io.grpc:grpc-netty:1.33.0")
+  runtime("io.netty:netty-codec-http2:4.1.53.Final") {
```

Comment:
These, and any upgrades of transitives, should be constraints rather than full fledged dependencies. That is, wrapped in a `constraints` block. The difference is that a constraint only affects versions of things already on the claspath, while this adds another version to the classpath for conflict resolution. This means if it's later fixed by the proper maintainer (such as by removing the dependency), the constraint has no effect but the dependency remains.